pallets/compliance-manager/src/benchmarking.rs

```diff
@@ -46,15 +84,32 @@ pub fn make_issuer<T: IdentityConfig + TestUtilsFn<AccountIdOf<T>>>(id: u32) ->
 ///   - It could have more complexity if `TrustedIssuer::trusted_for` is a vector but not on
 ///   benchmarking of add/remove. That could be useful for benchmarking executions/evaluation of
 ///   complience requiriments.
-pub fn make_issuers<T: IdentityConfig + TestUtilsFn<AccountIdOf<T>>>(s: u32) -> Vec<TrustedIssuer> {
-    (0..s).map(|i| make_issuer::<T>(i)).collect::<Vec<_>>()
+pub fn make_issuers<T: IdentityConfig + TestUtilsFn<AccountIdOf<T>>>(
+    s: u32,
+    clain_type_len: Option<usize>,
+) -> Vec<TrustedIssuer> {
+    (0..s)
+        .map(|i| make_issuer::<T>(i, clain_type_len))
+        .collect::<Vec<_>>()
```

Comment:
```suggestion
        .collect()
```
Inferred from return type.